tests/test_schema.py

```diff
@@ -291,6 +296,82 @@ def testJoin2(self):
         self.assertEqual(s1.join("a", "b", "c"), "a_b_c")
         self.assertEqual(s1.join("a", "b", "c", "d"), "a_b_c_d")
 
+    def _makeMapper(self, name1="a", name2="bb", name3="ccc", name4="dddd"):
```

Comment:
Please add a doc string and preferably make this a public method (name it `makeMapper` instead of `_makeMapper`)

Reply:
What advantage does making a non-`test` method in a test suite provide? Nothing will import it, so the only things calling it are module-local, so "public" vs. "private" shouldn't really matter.

Reply:
Fair enough. I guess it's a matter of taste. I tend to make these things public because I might end find I want to share them among tests (by subclassing or moving the code). But private works too.

I guess the important thing is that a user that is trying to understand the test will want to know what it does. So private or public, it deserves a full doc string.

Reply:
Does the docstring I pushed help?

Reply:
Yes. Thanks!